apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/Keys.cs

```diff
@@ -275,49 +255,56 @@ private V1.KeyRange ToProtobuf()
     /// </summary>
     public sealed class Key
     {
-        /// <summary>
-        /// The values of this key.
-        /// </summary>
-        internal ListValue KeyParts { get; }
+        internal SpannerParameterCollection Parameters { get; }
 
         /// <summary>
         /// Creates a key from the values in the parameter collection.
```

Comment:
Possibly expand the documentation to note that a reference is stored to this collection - changes to the collection will be visible when the key is used.